MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, it is possible to customize the appearance of the SfScheduler without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+ <Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary />
+            <ResourceDictionary>
+                <x:String x:Key="SfSchedulerTheme">CommonTheme</x:String> 
+                <Color x:Key="SfSchedulerNormalHeaderTextColor">Red</Color>
+                <Color x:Key="SfSchedulerNormalViewHeaderDayTextColor">Blue</Color>
+                <Color x:Key="SfSchedulerNormalViewHeaderDateTextColor">Blue</Color> 
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+	</tr>
+    
+    <tr>
+    <td>
+            SfSchedulerHoverColor    
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of scheduler when it is hovered.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+    <tr>
+    <td>
+            SfSchedulerNormalBackground     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of scheduler.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerDisabledNavigationArrowColor      
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of navigation arrow when it is disabled. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalMoreAppointmentIndicatorColor       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of appointment indicator day, week and work week view. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalSelectedCellBackground        
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background Color of selected cell in scheduler.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalTodayHighlightBrush         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of today hightlight.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfSchedulerNormalSelectedAppointmentBackground          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background Color of appointment layout while select week and work week view.  
```

Comment:
Color of appointment border when it is selected.

Reply:
Updated